From 281f93b29190c67c86c4403b96efdbac957a0162 Mon Sep 17 00:00:00 2001
From: rezso <rezso@rezso.net>
Date: Sun, 15 Nov 2020 00:11:49 +0100
Subject: [PATCH] Add missing include
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fix ' concurrency.h:606:10: error: ‘atomic_bool’ in namespace ‘std’ does not name a type; did you mean ‘atomic_load’? ' build error.
---
 src/concurrency.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/concurrency.h b/src/concurrency.h
index df806f1eae..ceca909e9a 100644
--- a/src/concurrency.h
+++ b/src/concurrency.h
@@ -67,6 +67,7 @@
     #endif
 #endif
 
+#include <atomic>
 #include <memory>
 #include <mutex>
 
